src/hash_types.rs

```diff
@@ -0,0 +1,65 @@
+// Rust Bitcoin Library
+// Written in 2014 by
+//   Andrew Poelstra <apoelstra@wpsoftware.net>
```

Comment:
I don't mind, but just so you know you can put your own name and date on new files, or "the rust-bitcoin developers" or whatever you want.